modules/logic_app/standard/main.tf

```diff
@@ -10,6 +10,6 @@ locals {
   resource_group   = var.resource_groups[try(var.settings.lz_key, var.settings.resource_group.lz_key, var.client_config.landingzone_key)][try(var.settings.resource_group.key, var.settings.resource_group_key)]
   storage_account  = var.storage_accounts[try(var.settings.lz_key, var.settings.storage_account.lz_key, var.client_config.landingzone_key)][try(var.settings.storage_account.key, var.settings.storage_account_key)]
   app_service_plan = var.app_service_plans[try(var.settings.app_service_plan.lz_key, var.settings.lz_key, var.client_config.landingzone_key)][try(var.settings.app_service_plan.key, var.settings.app_service_plan_key)]
-  app_settings     = try(var.app_settings, null)
+  app_settings     = var.external_app_settings == true ? null : try(var.app_settings, null)
```

Comment:
@ybensoussan dit was best guesstimate. Deze was iets anders dan de rest.

Reply:
voor consistentie zou ik hem zo doen inderdaad. Maar eigenlijk is hij bij deze niet nodig als ik het zo lees.

app_settings     = try(var.app_settings, null) -->als app_settings niet gezet is, dan null (leeg)